apps/image_augmentation/image_augmentation.cpp

```diff
@@ -170,34 +170,32 @@ int main(int argc, const char ** argv)
     {
         image0 = rocalResize(handle, input1, resize_w, resize_h, true);
     }
-    RocalImage image1 = rocalRain(handle, image0, false);
+    RocalTensor image1 = rocalRain(handle, image0, false);
 
-    RocalImage image11 = rocalFishEye(handle, image1, false);
+    RocalTensor image11 = rocalFishEye(handle, image1, false);
 
     rocalRotate(handle, image11, true, rand_angle);
 
     // Creating successive blur nodes to simulate a deep branch of augmentations
-    RocalImage image2 = rocalCropResize(handle, image0, resize_w, resize_h, false, rand_crop_area);;
+    RocalTensor image2 = rocalCropResize(handle, image0, resize_w, resize_h, false, rand_crop_area);;
     for(int i = 0 ; i < aug_depth; i++)
     {
         image2 = rocalBlurFixed(handle, image2, 17.25, (i == (aug_depth -1)) ? true:false );
     }
 
-    RocalImage image4 = rocalColorTemp(handle, image0, false, color_temp_adj);
+    // RocalTensor image4 = rocalColorTemp(handle, image0, true, color_temp_adj);
```

Comment:
Why this is commented out?

Reply:
The support in RPP for tensor is not present